cmd/csi/provisioner/command_test.go

```diff
@@ -3,11 +3,12 @@ package provisioner
 import (
 	"testing"
 
-	cmdManager "github.com/Dynatrace/dynatrace-operator/cmd/manager"
 	dtfake "github.com/Dynatrace/dynatrace-operator/pkg/api/scheme/fake"
 	dtcsi "github.com/Dynatrace/dynatrace-operator/pkg/controllers/csi"
 	"github.com/Dynatrace/dynatrace-operator/pkg/util/logger"
-	"github.com/Dynatrace/dynatrace-operator/test/mocks/cmd/config"
+	mocks "github.com/Dynatrace/dynatrace-operator/test/mocks/cmd/config"
+	mockedprovider "github.com/Dynatrace/dynatrace-operator/test/mocks/cmd/manager"
+	mockedmanager "github.com/Dynatrace/dynatrace-operator/test/mocks/sigs.k8s.io/controller-runtime/pkg/manager"
```

Comment:
IMO as package alias would be better to call them `providermock` and `managermock`

Reply:
And also change this one
`mocks "github.com/Dynatrace/dynatrace-operator/test/mocks/cmd/config"`
to 
`configmock "github.com/Dynatrace/dynatrace-operator/test/mocks/cmd/config"`